src/test/java/com/google/maps/PlacesApiIntegrationTest.java

```diff
@@ -0,0 +1,182 @@
+
+    // Phone numbers
+    assertNotNull(placeDetails.formattedPhoneNumber);
+    assertEquals(placeDetails.formattedPhoneNumber, "(02) 9374 4000");
+    assertNotNull(placeDetails.internationalPhoneNumber);
+    assertEquals(placeDetails.internationalPhoneNumber, "+61 2 9374 4000");
+
+    // Geometry
+    assertNotNull(placeDetails.geometry);
+    assertNotNull(placeDetails.geometry.location);
+    assertNotNull(placeDetails.geometry.location.lat);
+    assertEquals(placeDetails.geometry.location.lat, -33.866611, 0.001);
+    assertNotNull(placeDetails.geometry.location.lng);
+    assertEquals(placeDetails.geometry.location.lng, 151.195832, 0.001);
+
+    // URLs
+    assertNotNull(placeDetails.icon);
+    assertEquals(placeDetails.icon.toURI(),
+        new URI("https://maps.gstatic.com/mapfiles/place_api/icons/generic_business-71.png"));
+    assertNotNull(placeDetails.url);
+    assertEquals(placeDetails.url.toURI(), new URI("https://plus.google.com/111337342022929067349/about?hl=en-US"));
+    assertNotNull(placeDetails.website);
+    assertEquals(placeDetails.website.toURI(), new URI("https://www.google.com.au/about/careers/locations/sydney/"));
+
+    // Name
+    assertNotNull(placeDetails.name);
+    assertEquals(placeDetails.name, "Google");
+
+    // Opening Hours
+    assertNotNull(placeDetails.openingHours);
+    assertNotNull(placeDetails.openingHours.openNow);
+    assertNotNull(placeDetails.openingHours.periods);
+    assertNotNull(placeDetails.openingHours.weekdayText);
+    assertNotNull(placeDetails.utcOffset);
+
+    // Sydney can be either UTC+10 or UTC+11
+    assertTrue(placeDetails.utcOffset == 600 || placeDetails.utcOffset == 660);
+
+    // Photos
+    {
+      assertNotNull(placeDetails.photos);
+      Photo photo = placeDetails.photos[0];
+      assertNotNull(photo);
+      assertNotNull(photo.photoReference);
+      assertNotNull(photo.height);
+      assertNotNull(photo.width);
+      assertNotNull(photo.htmlAttributions);
+      assertNotNull(photo.htmlAttributions[0]);
+    }
+
+    // Reviews
+    {
+      assertNotNull(placeDetails.reviews);
+      PlaceDetails.Review review = placeDetails.reviews[0];
+      assertNotNull(review);
+      assertNotNull(review.authorName);
+      assertNotNull(review.authorUrl);
+      assertNotNull(review.language);
+      assertNotNull(review.rating);
+      assertNotNull(review.aspects);
+      PlaceDetails.Review.AspectRating aspect = review.aspects[0];
+      assertNotNull(aspect);
+      assertNotNull(aspect.rating);
+      assertNotNull(aspect.type);
+    }
+    // Place ID
+    assertNotNull(placeDetails.placeId);
+    assertEquals(placeDetails.placeId, GOOGLE_SYDNEY);
+    assertNotNull(placeDetails.scope);
+    assertEquals(placeDetails.scope, PlaceIdScope.GOOGLE);
+    assertNotNull(placeDetails.types);
+    assertEquals(placeDetails.types[0], "establishment");
+    assertNotNull(placeDetails.rating);
+    assertEquals(placeDetails.rating, 4.4, 1.0);
```

Comment:
This test seems particularly likely to change over time, causing flakiness without adding much value.